documentation/airport-guides/egkk.md

```diff
@@ -0,0 +1,235 @@
+### 1. About Gatwick Airport (LGW | EGKK)
+*Gatwick airport*, also known as *London Gatwick*, is a major international airport located in *Crawley, West Sussex*, 47.5km south of Central London.  
+*London Gatwick* is the second-busiest airport in the United Kingdom as well as the eighth-busiest airport in the world. The airport had also been holding the record as the world's busiest airport with only one operational runway at a time until 2017 when *Mumbai airport* overtook this record.  
+  
+  
+#### 1.1 Statistics
+* **Total passengers**: *46,576,473* (2019)
+* **Aircraft movements**: *284,987* (2019)
+* **Cargo**: *110,358 tonnes* (2019)
+  
+  
+### 2. Airspace
+In openScope, the airspace responsible for *Gatwick airport* is a combination of multiple sectors in reality. It encompasses large parts of the *London Terminal Control Area*, mainly to the south as the northern sectors are mainly used for traffic to and from *London Heathrow*, *London Luton* and *London Stansted*. The airspace extends from the surface up to maximum FL195, depending on the airspace section (see "Airspace Depiction"-view in openScope).  
+  
+#### 2.1 Sectors available in openScope
+The sim features a combination of these sectors:  
+
```

Comment:
It could be good to add in the specific TC sectors that are part of the simulated area. Looks to me like they're OCK, BIG, WILLO and TIMBA but I could be wrong.

Reply:
In the previous version (from two years ago) I provided more detailed information about the TC sectors but I couldnt find any reliable information to verify this. The only sectorization I found was for the London FIR high sectors. That's why I didn't include it anymore. The airspace is taken from the VATSIM sectorfile: the two sectors "LTC south-east" and "LTC south-west" (which are both combinations of multiple real sectors I think). I left that out because I think it's better to omit information than featuring information that is or might be not 100% correct. But if you have a good source for that, I would really like to add it further information about the TC sectors.

Reply:
Yeah... unfortunately the TC sectors aren't publicly available. Here's a diagram from 2008:
![image](https://user-images.githubusercontent.com/13125311/113780555-490c6580-9727-11eb-8368-b3e0e5b82ff5.png)
As you can see, the boundaries (both lateral and vertical) have changed somewhat, but the sectors are still the same ones afaik. @indianbhaji might know better as well (the VATUK discord will be happy to answer too I'm sure)

Reply:
@Fechulo I could verify this information with multiple sources (Eurocontrol Regional Chart ERC-10/L, Wikipedia (No source declared), ATC Frequency overviews, frequencies that are mentioned in the charts) and seems like the sectors are actually still the same ones.

Reply:
The "sector available in openScope" section has been updated accordingly and I decided to change the radio callsign for the APP to "London Control" instead of "Gatwick Director" because the initial contact of an inbound aircraft is with the responsible London Control sector and not the Director.